### PR TITLE
bnb get_address

### PR DESCRIPTION
**Pull request by BitHighlander**

### CODE REVIEW

File: keepkeyctl [hunk 1]
```diff
@@ -309,6 +309,11 @@ class Commands(object):
         address = self.client.cosmos_get_address(address_n, args.show_display)
         return address
 
+    def binance_get_address(self, args):
```
Comment (keepkeyjon) [inline keepkeyctl]: Needs a `.help` like here:
https://github.com/keepkey/python-keepkey/pull/94/files#diff-65792a4bbabaa290f0fa80d3394a2cc0R477

And `.args` like here:
https://github.com/keepkey/python-keepkey/pull/94/files#diff-65792a4bbabaa290f0fa80d3394a2cc0R552

File: tests/test_msg_binance_getaddress.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import keepkeylib.types_pb2 as proto_types
+import keepkeylib.messages_binance_pb2 as binance_proto
+from keepkeylib.client import CallException
+from keepkeylib.tools import parse_path
+
+DEFAULT_BIP32_PATH = "m/44h/118h/0h/0/0"
+
+class TestMsgBinanceGetAddress(common.KeepKeyTest):
+    def test_standard(self):
+        self.requires_firmware("6.3.0")
```
Comment (keepkeyjon) [inline tests/test_msg_binance_getaddress.py]: This should be `6.4.0`, since it's not in `6.3.0`.

Comment (keepkeyjon) [reply tests/test_msg_binance_getaddress.py]: (I'm about to push the corresponding bump in the firmware repo, FWIW)

File: tests/test_msg_binance_getaddress.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import unittest
+import common
+
+import keepkeylib.messages_pb2 as proto
+import keepkeylib.types_pb2 as proto_types
+import keepkeylib.messages_binance_pb2 as binance_proto
+from keepkeylib.client import CallException
+from keepkeylib.tools import parse_path
+
+DEFAULT_BIP32_PATH = "m/44h/118h/0h/0/0"
```
Comment (keepkeyjon) [inline tests/test_msg_binance_getaddress.py]: Stale?

File: tests/test_msg_binance_getaddress.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import unittest
+import common
+
+import keepkeylib.messages_pb2 as proto
+import keepkeylib.types_pb2 as proto_types
+import keepkeylib.messages_binance_pb2 as binance_proto
+from keepkeylib.client import CallException
```
Comment (keepkeyjon) [inline tests/test_msg_binance_getaddress.py]: Unused
```suggestion
```

File: tests/test_msg_binance_getaddress.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
+import unittest
+import common
+
+import keepkeylib.messages_pb2 as proto
+import keepkeylib.types_pb2 as proto_types
```
Comment (keepkeyjon) [inline tests/test_msg_binance_getaddress.py]: Unused
```suggestion
```

File: tests/test_msg_binance_getaddress.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
+import unittest
+import common
+
+import keepkeylib.messages_pb2 as proto
```
Comment (keepkeyjon) [inline tests/test_msg_binance_getaddress.py]: Unused
```suggestion
```

File: keepkeyctl [hunk 7]
```diff
@@ -554,6 +555,12 @@ class Commands(object):
         (('-d', '--show-display'), {'action': 'store_true', 'default': False}),
     )
 
+
+    binance_get_address.arguments = (
+        (('-n', '-address'), {'type': str, 'help': "BIP-32 path to source address", 'default': "m/44'/118'/0'/0/0"}),
```
Comment (keepkeyjon) [inline keepkeyctl]: default path is copy-pasta... that needs fixing

**Comment by markrypt0** [comment]
Resubmit if needed

**Pull request closed without merging**